docs/Users/Install.rst

```diff
@@ -153,6 +153,18 @@ If you want to install coala to an alternate location you can e.g. call
 options are documented on
 https://docs.python.org/3.3/install/#alternate-installation.
 
+.. note::
+
+    If you are using a proxy, follow these steps:
+
+    -  Set up your proxy.
+    -  Use ``sudo -E pip install coala``(-E flag takes in the existing
```

Comment:
I actually like the original more because preserving would rather mean for the invoking shell IMO. How about

    (the ``-E`` flag takes the existing environment variables into the ``sudo`` environment)

?